documentation/directory-structure.md

```diff
@@ -7,190 +7,259 @@ https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
+    - [ARM services: `resource-manager/<RPNS>` folder](#arm-services-resource-managerrpns-folder)
+    - [Data-plane services: `data-plane` folders](#data-plane-services-data-planeservice-folders)
   - [Service folder structure](#service-folder-structure)
+    - [Contents of a service folder](#contents-of-a-service-folder)
+    - [`<apiVersion>` folders](#apiVersion-folders)
   - [`specification/common-types`](#specificationcommon-types)
   - [Naming guidelines for `specification` folder contents](#naming-guidelines-for-specification-folder-contents)
-  - [About legacy, deprecated directory structure for services and service groups](#about-legacy-deprecated-directory-structure-for-services-and-service-groups)
-  - [Migrating from a singular service to service group in a legacy directory structure](#migrating-from-a-singular-service-to-service-group-in-a-legacy-directory-structure)
-  - [Deprecated directory structure and hand-written OpenAPI specs](#deprecated-directory-structure-and-hand-written-openapi-specs)
+  - [Legacy and deprecated patterns](#legacy-and-deprecated-patterns)
+  - [Current migration efforts](#current-migration-efforts)
+  - [Existing violations and historical context](#existing-violations-and-historical-context)
 
-# `specification` directory structure
+# Azure REST API Specification Directory Structure
 
-This article describes the directory structure / folder layout of the `specification` folder.
-You may be also interested in following:
-
-- [Specification index]
-- [Resource Provider list]
+This article provides a comprehensive guide to the recommended directory structure and folder layout for Azure REST API specifications within the `specification` folder. The structure outlined here is designed to support both ARM (Azure Resource Manager) and data-plane services while maintaining compatibility with both TypeSpec-based and OpenAPI-based API development workflows.
 
 > [!IMPORTANT]
-> The structure described in this article is recommended. There exist some exceptions for historical reasons.
-> These exceptions are strongly discouraged going forward.
+> The structure described in this article is the **recommended standard** for all new Azure services. While some existing services may follow legacy patterns for historical reasons, all new services must adopt this structure. We are actively working on migrating legacy patterns to align with these guidelines.
 
-## Key concepts
+## Key concepts and terminology
 
-The directory structure is a reflection of a few key concepts.
+To better understand the directory structure, it's essential to familiarize yourself with these key concepts as defined in the [glossary]:
 
-Familiarize yourself with the concepts of `service`, `service group`, `grouping directory` and `uniform versioning`
-as explained in the [glossary].
+- **Service**: A customer-facing Azure service (e.g., Azure Key Vault, Azure Container Registry)
```

Comment:
We should update the definition of service to match what is in the glossary / Azure Service Versioning documentation

Reply:
is this the same service name as defined on Service Tree?